backstory/src/main/java/com/google/sps/servlets/data/EntityFactory.java

```diff
@@ -0,0 +1,6 @@
+import com.google.appengine.api.datastore.Entity;
```

Comment:
Needs Google license. Applies to all classes (have mentioned it a couple times) that are missing it.